### PR TITLE
feat: add abbreviations to `hacker.abbreviation()` in the `en` locale

### PR DESCRIPTION
**Pull request by import-brain**
some logic gates, other acronyms and abbreviations

### CODE REVIEW

**Comment by xDivisionByZerox** [comment]
Aren't the "logic gates" just plain words? I'm questioning the decision to put them in the abbreviations file.

File: src/locales/en/hacker/abbreviation.ts [hunk 1]
```diff
@@ -28,4 +29,18 @@ export default [
...
   'JBOD',
+  'AND',
+  'OR',
+  'XOR',
+  'NOT',
+  'NAND',
+  'NOR',
+  'XNOR',
+  'DNS',
+  'ASCII',
```
Comment (xDivisionByZerox) [inline src/locales/en/hacker/abbreviation.ts:40]: What about other encodings? (UTF8, HEX, USC2, ...)

Comment (ST-DDT) [reply src/locales/en/hacker/abbreviation.ts:40]: Utf8 and hex yes.
IMO USC2 is probably not common enough.

How about TLS and API?

Comment (import-brain) [reply src/locales/en/hacker/abbreviation.ts:40]: Hmm, okay. I'll remove the logic gates and add TLS, API, and the two encodings.

**Review by ST-DDT — DISMISSED** [review]

**Review by Shinigami92 — COMMENTED** [review]
SSD!

**Comment by xDivisionByZerox** [comment]
> SSD!

What? We already have that.

**Comment by ST-DDT** [comment]
> > SSD!
> 
> What? We already have that.

Can you give me a line number?
I only see SDD (Same Day Delivery?).

**Comment by xDivisionByZerox** [comment]
> Can you give me a line number? I only see SDD (Same Day Delivery?).

Damn, your write. I literally read that as SSD...

Well in that case we can add `'HDD'` as well.

**Comment by ST-DDT** [comment]
Looks like your brain is melting just like mine 😉😅🥵

**Comment by ST-DDT** [comment]
Please either sort them alphabetically or sort them by their category as before.

File: src/locales/en/hacker/abbreviation.ts [hunk 2]
```diff
@@ -1,31 +1,43 @@
...
+  'RSS',
   'SAS',
-  'IB',
   'SCSI',
-  'JSON',
+  'SDD',
+  'SMS',
+  'SMTP',
+  'SQL',
+  'SSL',
```
Comment (Shinigami92) [inline src/locales/en/hacker/abbreviation.ts:33]:
```suggestion
  'SSD',
  'SSL',
```

**Review by ST-DDT — APPROVED** [review]

**Review by Shinigami92 — DISMISSED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Comment by import-brain** [comment]
I don't know if I'm doing this correctly or not, but the results that the seeded tests return seem to change on every run of the tests now that there are snapshots. I've manually changed them multiple times, but the received value from the tested function seems to change everytime. Also, updating the snapshot by pressing "u" after the tests run doesn't seem to do anything, what's up with that?

**Comment by Shinigami92** [comment]
You left the seededRuns object and didn't remove it

**Review by Shinigami92 — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Comment by import-brain** [comment]
> You left the seededRuns object and didn't remove it

Ahhh, I see. Thanks Shini :)

**Pull request merged**